### PR TITLE
Fetch connection status for production entities from manage

### PR DESCRIPTION
**Pull request by pablothedude**
The connection status for services could be determined based on
the entities on the production manage and by looking at the
'exclude from push' coin.

### CODE REVIEW

**Comment by pablothedude** [comment]
https://www.pivotaltracker.com/story/show/166027604

**Comment by pablothedude** [comment]
https://www.pivotaltracker.com/story/show/164241210

File: src/Surfnet/ServiceProviderDashboard/Application/Service/ServiceStatusService.php [hunk 1]
```diff
@@ -92,4 +92,44 @@ public function getEntityStatus(Service $service)
         // No published or drafted entities discovered, state "No"
         return Service::ENTITY_PUBLISHED_NO;
     }
+
+
+    /**
+     * - Status: "No" when no test entity, and no draft on test is present
```
Comment (MKodde) [inline src/Surfnet/ServiceProviderDashboard/Application/Service/ServiceStatusService.php]: Please update the doc block. It speaks of the test env. Where the prod env is applicable.

Comment (pablothedude) [reply src/Surfnet/ServiceProviderDashboard/Application/Service/ServiceStatusService.php]: Done, also renamed the method to `getConnectionStatus` to reflect the purpose of the method better.

**Comment by pablothedude** [comment]
Added some extra tests

**Pull request merged**